controllers/machinedeployment_controller_test.go

```diff
@@ -389,6 +389,21 @@ var _ = Describe("MachineDeployment Reconciler", func() {
 			return len(machineSets.Items)
 		}, timeout*5).Should(BeEquivalentTo(0))
 
+		By("Verifying MachineDeployment has correct Conditions")
+		Eventually(func() bool {
+			key := client.ObjectKey{Name: deployment.Name, Namespace: deployment.Namespace}
+			if err := testEnv.Get(ctx, key, deployment); err != nil {
+				return false
+			}
+			var availableCond bool
```

Comment:
why do we need `availableCond` at all? can't we just return true/false below?

Reply:
I saw this convention used in other the places and used it here as other conditions will be added and checked.

Reply:
Could you please point to other place where this is done so I can have a look? Regardless of where else it is used I see no value here.